pandas_datareader/google/options.py

```diff
@@ -0,0 +1,197 @@
+
+        Returns
+        -------
+        pandas.DataFrame
+            A DataFrame with requested options data.
+
+            Index:
+                Strike: Option strike, int
+                Expiry: Option expiry, Timestamp
+                Type: Call or Put, string
+                Symbol: Option symbol as reported on Google, string
+            Columns:
+                Last: Last option price, float
+                Chg: Change from prior day, float
+                Bid: Bid price, float
+                Ask: Ask price, float
+                Vol: Volume traded, int64
+                Open_Int: Open interest, int64
+                IsNonstandard: True if the the deliverable is not 100 shares, otherwise false
+                Underlying: Ticker of the underlying security, string
+                Underlying_Price: Price of the underlying security, float64
+                Quote_Time: Time of the quote, Timestamp
+
+        Notes
+        -----
+        Note: Format of returned data frame is dependent on Google and may change.
+
+        When called, this function will add instance variables named
+        calls and puts. See the following example:
+
+            >>> aapl = Options('aapl', 'google')  # Create object
+            >>> aapl.calls  # will give an AttributeError
+            >>> aapl.get_options()  # Get data and set ivars
+            >>> aapl.calls  # Doesn't throw AttributeError
+
+        Also note that aapl.calls and appl.puts will always be the calls
+        and puts for the next expiry. If the user calls this method with
+        a different expiry, the ivar will be named callsYYMMDD or putsYYMMDD,
+        where YY, MM and DD are, respectively, two digit representations of
+        the year, month and day for the expiry of the options.
+
+        """
+        d = self._load_data(expiry)
+        return self._process_data(d, expiry)
+
+    @property
+    def expiry_dates(self):
+        """
+        Returns a list of available expiry dates
+        """
+        try:
+            expiry_dates = self._expiry_dates
+        except AttributeError:
+            d = self._load_data(dt.datetime.today())
+            expiry_dates = d['expirations']
+        return map(lambda d: dt.date(d['y'], d['m'], d['d']), expiry_dates)
+
+    def _load_data(self, expiry):
+        url = self._OPTIONS_BASE_URL.format(sym=self.symbol, day=expiry.day,
+                                            month=expiry.month, year=expiry.year)
+        s = re.sub(r'(\w+):', '"\\1":', self._read_url_as_StringIO(url).read())
+        return json.loads(s)
+
+    def _process_data(self, jd, expiry):
+        """
+        Parse JSON data into the DataFrame
+
+        """
+        now = dt.datetime.now()
+
+        columns = ['Last', 'Bid', 'Ask', 'Chg', 'PctChg', 'Vol', 'Open_Int', 'Root', 'Underlying_Price', 'Quote_Time']
+        indexes = ['Strike', 'Expiry', 'Type', 'Symbol']
+        rows_list = []
```

Comment:
Do not pass empty lists. Simply `_process_row` should return the results as lists.